### PR TITLE
fix: updating beacons text and adding translations

### PR DESCRIPTION
**Pull request by adriawh**
### What have been done
- Updated the texts to be according to [Figma](https://www.figma.com/file/zdZwvobgpEWSagKt0tderx/App?type=design&node-id=18256-2778&mode=dev) and will resolve the comments in [Figma](https://www.figma.com/file/zdZwvobgpEWSagKt0tderx/App?type=design&node-id=18256-2778&mode=dev) when this is merged. 
- Added translations for the text for iOS.
- The announcements have been updated in firebase to match [Figma](https://www.figma.com/file/zdZwvobgpEWSagKt0tderx/App?type=design&node-id=18256-2778&mode=dev). These were only in staging, and I did not add them to prod.

### Note
The "new-norwegian" texts should probably be double checked by @marius-at-atb or @rosvik. (Nynorsk wasn't my strongest side in high school)

### CODE REVIEW

**Review by gorandalum — APPROVED** [review]

**Comment by adriawh** [comment]
Waiting for the Nynorsk review by @marius-at-atb before merging

File: src/translations/screens/ShareTravelHabits.ts [hunk 1]
```diff
@@ -38,35 +38,35 @@ const ShareTravelHabitsTexts = {
...
-        "Du gir oss løyve til å kjenne att bussar og busshaldeplassar i nærleiken. Dette gjer at vi kan analysere kvar du og andre kundar reiser, og til å sende deg relevante undersøkingar. Det vil hjelpe oss til å kunne betre reisetilbodet."
-      )
+        'Du gir oss lov til å gjenkjenne busser og bussholdeplasser i nærheten. Det gjør at vi kan analysere hvor du og andre kunder reiser, og til å sende deg relevante undersøkelser. Det vil hjelpe oss til å kunne forbedre ditt reisetilbud.',
+        'You give us permission to recognize buses and bus stops nearby. This enables us to analyze where you and other customers travel, and to send you relevant surveys. The data is used to improve our services to make them better for you.',
+        'Du gir oss løyve til å kjenne att bussar og busshaldeplassar i nærleiken. Dette gjer at vi kan analysere kvar du og andre kundar reiser, og til å sende deg relevante undersøkingar. Det vil hjelpe oss til å kunne betre ditt reisetilbod.',
```
Comment (marius-at-atb) [inline src/translations/screens/ShareTravelHabits.ts:43]:
```suggestion
        'Du gir oss løyve til å kjenne att bussar og busshaldeplassar i nærleiken. Dette gjer at vi kan analysere kvar du og andre kundar reiser, og til å sende deg relevante undersøkingar. Det vil hjelpe oss til å kunne betre reisetilbodet ditt .',
```

File: src/translations/screens/ShareTravelHabits.ts [hunk 2]
```diff
@@ -38,35 +38,35 @@ const ShareTravelHabitsTexts = {
...
-        "You give permission to use movement data for analysis and greater understanding of travel patterns, e.g. to distinguish between cycling, walking and driving.",
-        "Du gir løyve til å bruke rørsledata til analyse og større forståing av reisemønster, til dømes for å skilje mellom sykling, gåing og bilkøyring."
-      )
+        'Du gir oss lov til å bruke disse dataene for å lære mer om dine reisevaner. Vi kan for eksempel se om kunder velger å sykle, gå eller kjøre bil fremfor å busse. Det vil hjelpe oss til å kunne forbedre ditt reisetilbud.',
+        'You give us permission to use this data to learn more about your travel habits. For example, we can see whether customers choose to cycle, walk or drive a car rather than take the bus. The data is used to improve our services to make them better for you.',
+        'Du gir oss løyve til å bruke disse dataene for å læra meir om dine reisevaner. Vi kan for eksempel se om kunder veljer å sykle, gå eller kjøre bil framan å busse. Det vil hjelpe oss til å kunne forbetre ditt reisetilbod.',
```
Comment (marius-at-atb) [inline src/translations/screens/ShareTravelHabits.ts:67]:
```suggestion
        'Du gir oss løyve til å bruke desse dataene for å lære meir om reisevanane dine. Vi kan til dømes sjå om kundar vel å sykle, gå eller køyre bil heller enn å busse. Det vil hjelpe oss til å kunne forbetre reisetilbodet ditt.',
```

File: ios/nn.lproj/InfoPlist.strings [hunk 3]
```diff
@@ -1,5 +1,11 @@
-"NSLocationAlwaysAndWhenInUseUsageDescription" = "I tillegg til å kunne vise di posisjon i kart og til å planleggje reiser, vil vi få verdifull læring om kor du og andre kundar reiser. Det vil hjelpe oss til å kunne forbetre reisetilbodet.";
+"NSLocationAlwaysAndWhenInUseUsageDescription" = "I tillegg til å kunne vise di posisjon i kart og til å planleggje reiser, vil vi få verdifull læring om kor du og andre kundar reiser. Det vil hjelpe oss til å kunne forbetre ditt reisetilbod.";
```
Comment (marius-at-atb) [inline ios/nn.lproj/InfoPlist.strings:1]:
```suggestion
"NSLocationAlwaysAndWhenInUseUsageDescription" = "I tillegg til å kunne vise di posisjon i kart og til å planleggje reiser, vil vi få verdifull læring om kor du og andre kundar reiser. Det vil hjelpe oss til å kunne forbetre reisetilbodet ditt.";
```

File: ios/nn.lproj/InfoPlist.strings [hunk 4]
```diff
@@ -1,5 +1,11 @@
...
 
-"NSLocationAlwaysUsageDescription" = "I tillegg til å kunne vise di posisjon i kart og til å planleggje reiser, vil vi få verdifull læring om kor du og andre kundar reiser. Det vil hjelpe oss til å kunne forbetre reisetilbodet.";
+"NSLocationAlwaysUsageDescription" = "I tillegg til å kunne vise di posisjon i kart og til å planleggje reiser, vil vi få verdifull læring om kor du og andre kundar reiser. Det vil hjelpe oss til å kunne forbetre ditt reisetilbod.";
```
Comment (marius-at-atb) [inline ios/nn.lproj/InfoPlist.strings:3]:
```suggestion
"NSLocationAlwaysUsageDescription" = "I tillegg til å kunne vise di posisjon i kart og til å planleggje reiser, vil vi få verdifull læring om kor du og andre kundar reiser. Det vil hjelpe oss til å kunne forbetre reisetilbodet ditt.";
```

File: ios/nn.lproj/InfoPlist.strings [hunk 5]
```diff
@@ -1,5 +1,11 @@
...
 
 "NSLocationWhenInUseUsageDescription" = "Vi brukar posisjon til å vise di posisjon i kart og reisesøk, og til å finne haldeplassar og stader i nærleiken.";
+
+"NSBluetoothAlwaysUsageDescription" = "Du gir oss løyve til å kjenne att bussar og busshaldeplassar i nærleiken. Dette gjer at vi kan analysere kvar du og andre kundar reiser, og til å sende deg relevante undersøkingar. Det vil hjelpe oss til å kunne betre ditt reisetilbod.";
```
Comment (marius-at-atb) [inline ios/nn.lproj/InfoPlist.strings:7]:
```suggestion
"NSBluetoothAlwaysUsageDescription" = "Du gir oss løyve til å kjenne att bussar og busshaldeplassar i nærleiken. Dette gjer at vi kan analysere kvar du og andre kundar reiser, og til å sende deg relevante undersøkingar. Det vil hjelpe oss til å kunne betre reisetilbodet ditt.";
```

File: ios/nn.lproj/InfoPlist.strings [hunk 6]
```diff
@@ -1,5 +1,11 @@
...
+
+"NSBluetoothPeripheralUsageDescription" = "Du gir oss løyve til å kjenne att bussar og busshaldeplassar i nærleiken. Dette gjer at vi kan analysere kvar du og andre kundar reiser, og til å sende deg relevante undersøkingar. Det vil hjelpe oss til å kunne betre ditt reisetilbod.";
```
Comment (marius-at-atb) [inline ios/nn.lproj/InfoPlist.strings:9]:
```suggestion
"NSBluetoothPeripheralUsageDescription" = "Du gir oss løyve til å kjenne att bussar og busshaldeplassar i nærleiken. Dette gjer at vi kan analysere kvar du og andre kundar reiser, og til å sende deg relevante undersøkingar. Det vil hjelpe oss til å kunne betre reisetilbodet ditt.";
```

File: ios/nn.lproj/InfoPlist.strings [hunk 7]
```diff
@@ -1,5 +1,11 @@
...
+
+"NSMotionUsageDescription" = "Du gir oss løyve til å bruke disse dataene for å læra meir om dine reisevaner. Vi kan for eksempel se om kunder veljer å sykle, gå eller kjøre bil framan å busse. Det vil hjelpe oss til å kunne forbetre ditt reisetilbod.";
```
Comment (marius-at-atb) [inline ios/nn.lproj/InfoPlist.strings:11]:
```suggestion
"NSMotionUsageDescription" = "Du gir oss løyve til å bruke disse dataene for å læra meir om dine reisevaner. Vi kan for eksempel se om kunder veljer å sykle, gå eller kjøre bil framan å busse. Det vil hjelpe oss til å kunne forbetre reisetilbodet ditt.";
```

**Review by marius-at-atb — APPROVED** [review]

**Pull request merged**